catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/NewProjectDialogFragment.java

```diff
@@ -120,12 +120,17 @@ void showOrientationDialog(String projectName, boolean createEmptyProject) {
 		dialog.show(getFragmentManager(), OrientationDialogFragment.TAG);
 	}
 
-	void createDroneProject(String name, boolean jumpingSumo) {
+	void createDroneProject(String name, boolean isJumpingSummoProject) {
 		try {
-			ProjectManager.getInstance().initializeNewProject(name, getActivity(), false, true,
-					false, false, jumpingSumo);
+			if (isJumpingSummoProject) {
+				ProjectManager.getInstance().initializeNewProject(name, getActivity(), false, false,
```

Comment:
maybe this would be a goot time to finally get rid of all those boolean flags in initialize project and just create an instance of the desired DefaultProjectCreator and pass this in, instead of boolean flags, and then the Enum in defaultProjectCreator.

Reply:
actually it's a bad time for me, I would work on what you asked for after being done with Visual Placement. could you please create a ticket for it and assign it to me.
thanks